dockers/piped-base/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.10.1
 
-ENV PIPED_BIN_DIR=/usr/local/piped
+ENV PIPED_BIN_DIR=/home/pipecd
```

Comment:
nit: `/home/pipecd/tools`

Reply:
Because we will have `.ssh` directory in this too.

Reply:
Right, it should be aggregated into one place.